### ISSUE TITLE
Issue status better highlight

### ISSUE THREAD

**Issue by Yakubina** [andry-virvich/jira#17]
Hi! I propose to highlight a status of issue in the buttons bar another way.

![image](https://user-images.githubusercontent.com/6073766/34385856-444d8252-eb2e-11e7-9cd5-a4131221315d.png)
Now status is showing only for "in progress" issues, 

More informative will be making it this way: 
![image](https://user-images.githubusercontent.com/6073766/34385869-6430f18a-eb2e-11e7-8d5a-a528627ebc7d.png)
"Done" is highlighted as an active status of issue

**Comment by avyrvich** [comment]
@Yakubina this actions buttons depend on transitions available for the ticket and statusCategory this transitions change status to:

````

  {
    "self": "https://jira.atlassian.com/rest/api/2/statuscategory/1",
    "id": 1,
    "key": "undefined",
    "colorName": "medium-gray",
    "name": "No Category"
  },
  {
    "self": "https://jira.atlassian.com/rest/api/2/statuscategory/2",
    "id": 2,
    "key": "new",
    "colorName": "blue-gray",
    "name": "To Do"
  },
  {
    "self": "https://jira.atlassian.com/rest/api/2/statuscategory/4",
    "id": 4,
    "key": "indeterminate",
    "colorName": "yellow",
    "name": "In Progress"
  },
  {
    "self": "https://jira.atlassian.com/rest/api/2/statuscategory/3",
    "id": 3,
    "key": "done",
    "colorName": "green",
    "name": "Done"
  }
]
````

so instead I'd highlight current status in attributes list for an issue:
![image](https://user-images.githubusercontent.com/891324/34388688-f54e6a2a-eae9-11e7-8448-ca5947be7974.png)

**Comment by avyrvich** [comment]
just released updated UI in 0.1.6

**Comment by Yakubina** [comment]
@andry-virvich , thank you! It looks much better!)
